src/compiler/parser/parser.rs

```diff
@@ -207,7 +207,7 @@ impl std::convert::TryInto<DisplayableColumn> for ColumnAttributes {
         }
         Ok(DisplayableColumn {
             name: self.name,
-            base: self.base.get().cloned().unwrap_or(Base::Dec),
+            base: self.base.get().cloned().unwrap_or(Base::Hex),
```

Comment:
This could be replaced by implementing `Defautl` for `Base` and using `unwrap_or_default()` everywhere a similar use case appears.